CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ option(TESTING      "Build and run test suite"                    ON )
 option(CLANG_FORMAT "Enable clang-format target"                  ON )
 option(CLANG_TIDY   "Enable clang-tidy checks during compilation" OFF)
 option(COVERAGE     "Enable generation of coverage info"          OFF)
+option(EMBEDDINGS   "Embed developers assets"                     ON)
```

Comment:
I think this option name isn't very descriptive. I'd suggest to go for for specific one, like EMBED_DEFAULT_SPEC.

Reply:
But it embeings not only spec, but key additionally